addons/website_sale/static/src/js/website_sale_utils.js

```diff
@@ -1,6 +1,43 @@
 odoo.define('website_sale.utils', function (require) {
 'use strict';
 
+const wUtils = require('website.utils');
+
+const cartHandlerMixin = {
+    getRedirectOption() {
+        const html = document.documentElement;
+        this.stayOnPageOption = html.dataset.add2cartRedirect !== '0';
+    },
+    getCartHandlerOptions(ev) {
+        this.isBuyNow = $(ev.currentTarget).attr('id') === 'buy_now';
```

Comment:
since it become an helper in utils to be re-usable.
We should probably use it with a class to support multi buy now button (eg. comparison) on a same page.
What you think about ?